sound/soc/sof/intel/hda-loader.c

```diff
@@ -344,6 +344,21 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 		goto cleanup;
 	}
 
+	/*
+	 * When soundwire is in clock stop state, slave will wake up
+	 * master for some events like jack plugged and wakeen interrupt
+	 * will be triggered by audio pci device and routed to PME if pci
```

Comment:
@plbossart this change seems to confusing to follow(probably because I am not very familiar with the way SDW works) but here's my concern. 
The above statement says that "wakeen interrupt  will be triggered by audio pci device". Is this really true? Isnt the wakeen interrupt triggered by the glue hardware that has received the jack detection event? Which in turn resumes the SOF PCI device? In which case, we should handle this event in the resume callback instead of when we boot the firmware? 
Maybe I am off track but I thought I'll ask anyway as this is quite different from how we handle the WAKEEN feature for HDA codecs.

Reply:
And oh the commit title prefix needs to be fixed as well

Reply:
reworded everything and fixed title